package.json

```diff
@@ -36,11 +36,12 @@
     "db:migrate-down": "node ./src/server/database/migrate.babel.js down",
     "db:migrate-down-all": "node ./src/server/database/migrate.babel.js down --all",
     "debug:stripe": "node ./src/server/billing/debug.babel.js",
+    "flow": "flow",
```

Comment:
@dan-f seems we need to have a `.flowconfig` for the build to pass? Digging in deeper…

Reply:
Oh shoot. I thought I added that. Hold on, I've got it.